features/step_definitions/basic_steps.rb

```diff
@@ -9,3 +9,15 @@
 When(/^I click "([^"]*)"$/) do |button|
   click_link_or_button button
 end
+
+Given(/^the following restaurant exists$/) do
+   @restaurant = create(:restaurant)
```

Comment:
Oof. Ok, look at how you use these steps in other features (I saw this in Jennifer's code yesterday) or how you did it in the photo market. It's ok to say `Given a restaurant exists` - but you want to keep this step definition, written as-is, for allowing the creation of multiple restaurants with names you can look for. Same thing with menu below.